### ISSUE TITLE
Consider adding biber to pre-installed packages

### ISSUE THREAD

**Issue by evilstiefel** [blang/latex-docker#10]
Thanks for this image, works just fine! I've started using it to kick off automatic builds of the PDF and publishing it using Gitlab-CI.

Since this image now is even sort of officially mentioned in the template for such tasks in the Gitlab documentation, why not make it more useable for biblatex-workflows by adding biber via apt?

Otherwise, any compilation will fail with the missing dependency.

**Comment by janstuemmel** [comment]
duplicate #5 

i have created one with biber installed https://github.com/janstuemmel/docker-latex

just change your yml file to:
```yml
image: janstuemmel/docker-latex

build:
  script:
    - latexmk -pdf
  artifacts:
    paths:
      - "*.pdf"
```

**Comment by flungo** [comment]
Unless there is widespread demand for a package I agree that you should either create or find an extended image like @janstuemmel suggested or in the case of GitLab CI usage, add the installation of the packages you need to your `before_script` in the `.gitlab-ci.yml`. See the [before_script](https://docs.gitlab.com/ce/ci/yaml/#before_script) reference.

**Comment by evilstiefel** [comment]
I did even before I created the issue. I only mentioned it since Gitlab references this image as the example for building latex-based pipelines and more people might choose this package as their default. I get that this was never the intention when the image was created, but just wanted to mention it.

Anyway, adding biber to the image has no downsides as far as I can see (next to a slight increase in size) and BibLaTeX is widely used and regarded by some as the successor of BibTeX.
